test/DurableTask.Netherite.AzureFunctions.Tests/TargetBasedScalingTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.Netherite.AzureFunctions.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using DurableTask.Core;
+    using DurableTask.Netherite.Scaling;
+    using DurableTask.Netherite.Tests;
+    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+    using Microsoft.Azure.WebJobs.Host.Scale;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using Xunit;
+    using Xunit.Abstractions;
+    using static DurableTask.Netherite.Scaling.ScalingMonitor;
+
+    public class TargetBasedScalingTests : IntegrationTestBase
+    {
+        readonly ILoggerFactory loggerFactory;
+        readonly Mock<NetheriteMetricsProvider> metricsProviderMock;
+        readonly Mock<IOrchestrationService> orchestrationServiceMock;
+
+        public TargetBasedScalingTests(ITestOutputHelper output) : base(output)
+        {
+            this.loggerFactory = new LoggerFactory();
+            var loggerProvider = new XunitLoggerProvider();
+            this.loggerFactory.AddProvider(loggerProvider);
+
+            this.orchestrationServiceMock = new Mock<IOrchestrationService>(MockBehavior.Strict);
+
+            this.metricsProviderMock = new Mock<NetheriteMetricsProvider>(
+                MockBehavior.Strict,
+                null,
+                new ConnectionInfo());
+        }
+
+        [Theory]
+        [InlineData(10, 10, 2)]
+        [InlineData(20, 20, 2)]
```

Comment:
@sebastianburckhardt, can you help provide expected input and output values to test the target based scaling algorithm? The inputs for this test are `maxConcurrentTaskActivityWorkItems` and `maxConcurrentTaskOrchestrationWorkItems`.